examples/example-gateway/build/clients/baz/baz_handler.go

```diff
@@ -0,0 +1,87 @@
+// Code generated by zanzibar
```

Comment:
With all the changes already in part3, it's easier to work this out over there..., so fix by https://github.com/uber/zanzibar/pull/57/commits/39bfe645dce529a7334439c5d821bff6dea11883 and https://github.com/uber/zanzibar/pull/57/commits/8cb587ea117e6a0a70aa9576b6d9326048785416